ts/dx.all.d.ts

```diff
@@ -6326,18 +6338,34 @@ declare module DevExpress.viz {
     }
     /** An object defining a gauge indicator of the rectangle type. */
     export interface linearRectangle extends CommonIndicator {
+        /** Specifies the distance between the indicator and the invisible scale line. */
+        offset?: number;
         /** Specifies the width of an indicator in pixels. */
         width?: number;
     }
+    /** An object that defines a gauge indicator of the circle type. */
+    export interface linearCircle extends CommonIndicator {
+        /** Specifies the distance between the indicator and the invisible scale line. */
+        offset?: number;
+    }
     /** An object defining a gauge indicator of the rhombus type. */
     export interface linearRhombus extends CommonIndicator {
+        /** Specifies the distance between the indicator and the invisible scale line. */
+        offset?: number;
         /** Specifies the width of an indicator in pixels. */
         width?: number;
     }
+    /** An object that defines a gauge indicator of the rangeBar type. */
+    export interface linearRangeBar extends CommonIndicator {
+        /** Specifies the distance between the indicator and the invisible scale line. */
+        offset?: number;
+    }
     /** An object that defines a gauge indicator of the triangleMarker type. */
     export interface linearTriangleMarker extends CommonIndicator {
         /** Specifies the indicator length. */
         length?: number;
+        /** Specifies the distance between the indicator and the invisible scale line. */
```

Comment:
What's that? Not referenced in PR's title

Reply:
It is a side effect of the experiment with TS in charts sources. Yes, I agree that it should be part of another PR.